code/scripting/api/objs/shipclass.cpp

```diff
@@ -270,6 +270,19 @@ ADE_VIRTVAR(AfterburnerFuelMax, l_Shipclass, "number", "Afterburner fuel capacit
 	return ade_set_args(L, "f", Ship_info[idx].afterburner_fuel_capacity);
 }
 
+ADE_VIRTVAR(ScanTime, l_Shipclass, "number", "Ship scan time", "number", "Time required to scan, or 0 if handle is invalid. This propery is read-only")
```

Comment:
Read-only properties should have a nullptr instead of the first type-string (as the first type string indicates the data type the variable can be set to in the docs).
So something like ``ADE_VIRTVAR(ScanTime, l_Shipclass, nullptr, "Ship scan time", "number", "Time required to scan, or 0 if handle is invalid. This propery is read-only")``

Reply:
Done